Resources/Prototypes/Floof/Interactions/noop_interactions.yml

```
@@ -0,0 +1,29 @@
+- type: Interaction
+  id: Kiss
+  parent: [BaseGlobal, BaseHands]
+  priority: 2
+  #icon: /Textures/Interface/Actions/hug.png
+  delay: 0.7
+  range: {max: 1}
+  hideByRequirement: true
+  requirement:
+    !type:MobStateRequirement
+    inverted: true
```

You don't want this to be usable on dead bodies. Probably. 
```suggestion
    allowedStates: [Alive, Critical]
```

---

:( no, there are people who are going to want that